src/modes/static.js

```diff
@@ -0,0 +1,11 @@
+module.exports = function(ctx) {
+  return {
+    stop: function() {
+      ctx.map.doubleClickZoom.enable();
```

Comment:
This can use the `doubleClickZoom` lib function: https://github.com/mapbox/mapbox-gl-draw/blob/master/src/lib/double_click_zoom.js